src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -26,29 +26,29 @@ limitations under the License.
     <!-- TODO: Add feature map once it is supported -->
     <description>This cluster provides an interface for controlling the current Channel on a device.</description>
     <attribute side="server" code="0x0000" define="CHANNEL_LIST" type="ARRAY" entryType="ChannelInfo" length="254" writable="false" optional="false">channel list</attribute>
```

Comment:
Again, the attribute names are off... Followup.

Reply:
https://github.com/project-chip/connectedhomeip/issues/13213